holoviews/plotting/bokeh/element.py

```diff
@@ -538,17 +538,44 @@ def __init__(self, element, plot=None, **params):
 
     def initialize_plot(self, ranges=None, plot=None, plots=None):
         self.mplplot.initialize_plot(ranges)
-        plot = mpl.to_bokeh(self.mplplot.state)
+
```

Comment:
Ok great.

There probably isn't much value in having Bokeh wrap matplotlib as you will always be limited by the speed of rendering on the matplotlib side. In general nesting backends probably doesn't make sense and it is probably best to simply state that a particular element isn't supported by a given backend.